infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -165,20 +160,8 @@
                     + "              Show this screen\n"
                     + " --version                                                                "
                     + "              Show version\n";
-    public static boolean serverRunning = false;
-    @Getter
-    private static SequencerServer sequencerServer;
-    @Getter
-    private static LayoutServer layoutServer;
-    @Getter
-    private static LogUnitServer logUnitServer;
-    @Getter
-    private static ManagementServer managementServer;
-    private static NettyServerRouter router;
+
     private static ServerContext serverContext;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "serverContext" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)